packages/env-tests/src/shared/attestation.ts

```diff
@@ -1,71 +1,17 @@
 import { notEmpty } from '@celo/base'
-import { Address, CeloTransactionParams } from '@celo/connect'
 import { ContractKit } from '@celo/contractkit'
-import {
-  ActionableAttestation,
-  AttestationsWrapper,
-  getSecurityCodePrefix,
-} from '@celo/contractkit/lib/wrappers/Attestations'
 import { OdisUtils } from '@celo/identity'
 import { AuthSigner, OdisContextName } from '@celo/identity/lib/odis/query'
 import { PhoneNumberUtils } from '@celo/phone-utils'
-import { AttestationRequest } from '@celo/phone-utils/lib/io'
-import { concurrentMap, sleep } from '@celo/utils/lib/async'
-import { AttestationUtils } from '@celo/utils/lib/attestations'
 import Logger from 'bunyan'
-import { sample } from 'lodash'
 import moment from 'moment'
 import { Twilio } from 'twilio'
 
-const POLLING_WAIT = 300
-
 export type RequestAttestationError =
```

Comment:
is this file (`/env-tests/src/shared/attestation.ts`) used anywhere or can it be deleted?